pylint/checkers/base/name_checker/checker.py

```diff
@@ -516,9 +535,10 @@ def _should_exempt_from_invalid_name(node):
         regexp = self._name_regexps[node_type]
         match = regexp.match(name)
 
```

Comment:
Was it handled in `` _is_multi_naming_match`` ? We could remove the handling there if that's the case.

Reply:
Yes, but the assertion was handled as well. So I chose to add an ignore instead.